helper_apps/MediaFileChunker/helper.js

```diff
@@ -28,6 +28,18 @@ async function deleteTempPath(path) {
     }
 }
 
+function ensureEncoded(url) {
+    try {
+        const decodedUrl = decodeURI(url);
+        if (decodedUrl === url) {
```

Comment:
Is this part necessary?  decodeURI will just throw if the URI is not encoded right?  So if it doesn't throw you just have a decoded URI which you probably don't need to re-encode?

Reply:
knuth got me

Reply:
i think knuth was right after all, it double encodes otherwise